modern/src/runtime/AnimatedLayer.ts

```diff
@@ -93,13 +184,11 @@ class AnimatedLayer extends Layer {
   }
 
   getstartframe() {
-    unimplementedWarning("getstartframe");
-    return;
+    return this.attributes.start;
   }
 
   getendframe() {
-    unimplementedWarning("getendframe");
```

Comment:
A commit I recently merged will make it an eslint error to have method that's missing a return type and does not call `unimplementedWarning`. Rebasing should show that. It should be fixable:

`yarn lint --fix`

Reply:
Awesome, thanks!